src/common/chess-engine.ts

```diff
@@ -232,6 +235,17 @@ export class ChessEngine {
      * @returns the move that was made.
      */
     makeMove(move: Move): Move {
+        //const yay = this.chess.history[this.chess.history.length-1];
+        //if(yay!==undefined && !(move.from === yay.from || move.to === yay.to))
+        this.chess.move(move);
+        return move;
+    }
+
+    /**
+     * Makes a move on the chessboard.
+     * @returns the move that was made.
+     */
+    move(move: string): string {
```

Comment:
this method isn't used anywhere, why is it included?

Reply:
so true bestie